src/applications/disability-benefits/all-claims/config/4192/index.js

```diff
@@ -27,78 +26,72 @@ const isUploading = formData =>
 
 // const isExiting = formData => _.get(formData, 'view:upload4192Choice.view:sendRequests', false);
 
-export default function() {
-  let configObj = {};
-  if (!environment.isProduction()) {
-    configObj = {
-      // Intro
-      pastEmploymentFormIntro: {
-        path: 'past-employment-walkthrough-choice',
-        depends: needsToEnterUnemployability,
-        uiSchema: pastEmploymentFormIntro.uiSchema,
-        schema: pastEmploymentFormIntro.schema,
-        onContinue: captureEvents.pastEmploymentFormIntro,
-      },
-      // Form Tutorial (multiple pages)
-      instructionalPart1: {
-        path: '4192-instructions-part-1',
-        depends: showFormTutorial,
-        uiSchema: instructionalPart1.uiSchema,
-        schema: instructionalPart1.schema,
-      },
-      instructionalPart2: {
-        path: '4192-instructions-part-2',
-        depends: showFormTutorial,
-        uiSchema: instructionalPart2.uiSchema,
-        schema: instructionalPart2.schema,
-      },
-      instructionalPart3: {
-        path: '4192-instructions-part-3',
-        depends: showFormTutorial,
-        uiSchema: instructionalPart3.uiSchema,
-        schema: instructionalPart3.schema,
-      },
-      // Download
-      pastEmploymentFormDownload: {
-        path: 'past-employment-download',
-        depends: isDownloading,
-        uiSchema: pastEmploymentFormDownload.uiSchema,
-        schema: pastEmploymentFormDownload.schema,
-      },
-      // Upload
-      pastEmploymentFormUpload: {
-        path: 'past-employment-form-upload',
-        depends: isUploading,
-        uiSchema: pastEmploymentFormUpload.uiSchema,
-        schema: pastEmploymentFormUpload.schema,
-      },
-      // ***Below page comments for when logic is added in***
-      // Conditional Options (Intro page again if A and not A && B && C)
-      // Form Tutorial (multiple pages)
-      // Download
-      // Upload
-      // Conditional Options (Intro page again if A and not A && B && C)
-      // Form Tutorial (multiple pages)
-      // Download
-      // Upload
-      // Conditional Options (Intro page again if A and not A && B && C)
-      // ***Above page comments for when logic is added in***
-      // Exit
-      conclusion4192: {
-        title: 'Conclusion 4192',
-        path: 'disabilities/conclusion-4192',
-        depends: needsToEnterUnemployability,
-        uiSchema: {
-          'ui:title': ' ',
-          'ui:description':
-            'Thank you for taking the time to answer our questions. The information you provided will help us process your claim.',
-        },
-        schema: {
-          type: 'object',
-          properties: {},
-        },
-      },
-    };
-  }
-  return configObj;
-}
+export default {
+  // Intro
+  pastEmploymentFormIntro: {
+    path: 'past-employment-walkthrough-choice',
```

Comment:
Hmm, I don't think we've thought that far ahead. These two forms were added over a year ago by a different team. And there aren't any unit or e2e tests for them yet. So there is still a lot of testing to do. I actually wonder if we should hold off on this significant change to 526 in case it breaks existing forms in progress.

I would prefer to make the change I suggested in Slack - https://dsva.slack.com/archives/CBU0KDSB1/p1614877014109600 - about having the `depends` function have access to feature toggles and control the view.